packages/react-dom/src/client/ReactDOMFiberComponent.js

```diff
@@ -471,9 +444,9 @@ export function setInitialProperties(
     case 'video':
     case 'audio':
       // Create listener for each media event
-      for (const event in mediaEvents) {
-        if (mediaEvents.hasOwnProperty(event)) {
-          trapBubbledEvent(event, mediaEvents[event], domElement);
+      for (const event in mediaEventTypes) {
+        if (mediaEventTypes.hasOwnProperty(event)) {
+          trapBubbledEvent(event, mediaEventTypes[event], domElement);
```

Comment:
`trapBubbledEvent` is kind of a confusing name now since it's attaching the listener directly to the event target in some cases.

Reply:
Agreed. It's more like "attach this event locally to emulate bubbling"